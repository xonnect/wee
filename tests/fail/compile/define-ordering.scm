(define foo bar)

(define (bar) 3)
