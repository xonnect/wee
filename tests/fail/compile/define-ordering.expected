*** PICOBIT ERROR -- variable referenced before its definition: bar at #f:#f
